src/Modules/StatCan.OrchardCore.Hackathon/Migrations.cs

```diff
@@ -0,0 +1,39 @@
+using OrchardCore.ContentManagement.Metadata;
+using OrchardCore.ContentManagement.Metadata.Settings;
+using OrchardCore.Data.Migration;
+
+namespace StatCan.OrchardCore.Hackathon
+{
+    public class Migrations : DataMigration
```

Comment:
Maybe this should be in the theme migrations

Reply:
I had started moving the FIP into the theme when I remembered that I had moved it out because I want the Vuetify theme to not contain Hackathon or StatCan related components. Are we sure we want the FIP to be inside the theme?

Reply:
Ah ok, maybe a FIP module would be best. StatCan.OrchardCore.FIP